### PR TITLE
Add announcements for week 15

### PR DESCRIPTION
**Pull request by marians**
To be sent out on Friday April 16, 14:00 CEST.

Please commit your announcements by then. Thank you!

### CODE REVIEW

**Comment by marians** [comment]
FYI @cokiengchiara @MartaD @alex-dabija

File: announcements/2021/week15.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## General
+
+We made a mistake, and we’re here to apologize. In our user interfaces, we previously indicated that Kubernetes v1.18 would run out of support (reach end of life, EOL) by April 8. This was a mistake. The **EOL date for v1.18 is April 14** (two days ago) after the Kubernetes project meanwhile had set it to April 30. Sorry for the confusion this may have caused!
+
+## Managed apps
+
+[NGINX IC v1.16.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.16.0/) fixes the validation of CPU requests and limits. It also updates the controller container image to v0.45.0, addressing OpenSSL CVEs.
```
Comment (cokiengchiara) [inline announcements/2021/week15.md:11]: @ubergesundheit does it address a CVE?

**Review by cokiengchiara — APPROVED** [review]

File: announcements/2021/week15.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# :zap: This week in provider-independent announcements (April 16, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## General
+
+We made a mistake, and we’re here to apologize. In our user interfaces, we previously indicated that Kubernetes v1.18 would run out of support (reach end of life, EOL) by April 8. This was a mistake. The **EOL date for v1.18 is April 14** (two days ago). Sorry for the confusion this may have caused!
+
+Our KVM team is working hard to provide a release with Kubernetes v1.19 soon.
```
Comment (marians) [inline announcements/2021/week15.md:9]: @cornelius-keller Addressing expectations here for team Rocket, as talked about on Slack.

Comment (cornelius-keller) [reply announcements/2021/week15.md:9]: Looks good to me, thx

**Review by cornelius-keller — APPROVED** [review]

**Pull request merged**